Orange/widgets/unsupervised/owmds.py

```diff
@@ -216,6 +216,8 @@ def __init__(self):
         g = self.graph.gui
         box = g.point_properties_box(self.controlArea)
         self.models = g.points_models
+        self.models[2].order = \
+            self.models[2].order[:1] + ("Stress", ) + self.models[2].order[1:]
```

Comment:
This modifies a model that is then used by all widgets. Copy the model before modifying it.